Sources/Public/CalendarView.swift

```diff
@@ -127,6 +116,7 @@ public final class CalendarView: UIView {
       guard
         newValue.size != bounds.size,
         scrollToItemContext == nil,
+        !scrollView.isDragging,
```

Comment:
This works around an issue where the calendar's bounds is changing at the same time that a user is dragging on the scroll view. This causes a weird feeling interaction, so better to guard against it.

Reminder that the point of this `bounds` `willSet` override is to do some bookkeeping so that we can maintain our scroll position after the bounds changes (like during rotation). If the user has their finger dragging on the calendar / is actively interacting with the calendar, then we don't need to do anything extra here.